include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
-    /// @replaces dnf:dnf/package.py:method:Package.localPkg(self)
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
+    // @replaces dnf:dnf/package.py:method:Package.localPkg(self)
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
     //bool is_local() const;
 
     /// TODO get_from_repo_id - requires swdb
     /// For an installed package, return repoid of repo from the package was installed.
     /// For an available package, return an empty string.
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.ui_from_repo
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_origin(DnfPackage * pkg)
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.ui_from_repo
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_origin(DnfPackage * pkg)
     // void get_from_repo_id() const;
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.hdr_end
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_hdr_end(DnfPackage * pkg)
-    unsigned long long get_hdr_end() const;
-
-    /// @replaces dnf:dnf/package.py:attribute:Package.installtime
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
+    /// @return The unix timestamp (seconds since epoch) when a package was installed.
+    ///         Return 0 if the package is not installed.
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.installtime
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
     unsigned long long get_install_time() const;
 
-    /// @brief Media number for the package
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.medianr
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_medianr(DnfPackage * pkg)
-    unsigned long long get_media_number() const;
-
-    /// @brief The rpmdb ID for the package
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.rpmdbid
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_rpmdbid(DnfPackage * pkg)
+    /// @return The rpmdb database id (primary key) of the installed RPM package
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.rpmdbid
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_rpmdbid(DnfPackage * pkg)
     unsigned long long get_rpmdbid() const;
 
-    /// TODO get_repo - requires Repo
-    /// @replaces dnf:dnf/package.py:attribute:Package.repo
-    /// @replaces dnf:dnf/package.py:attribute:Package.repoid
-    /// @replaces dnf:dnf/package.py:attribute:Package.reponame
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_repo(DnfPackage * pkg)
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_reponame(DnfPackage * pkg)
+    /// @return Repo object that represents the repository the package is part of.
```

Comment:
Do you think this is better?

> ... the repository the package **belongs to**

Also, we don't end these with a period in the other docstrings, keep it consistent?

Reply:
Fixed.